components/admin_console/admin_definition.jsx

```diff
@@ -4685,7 +4685,6 @@ const AdminDefinition = {
             title: t('admin.sidebar.gitlab'),
             title_default: 'GitLab',
             isHidden: it.any(
-                it.licensed,
```

Comment:
This page is related to oauth and is available for unlicensed users. We don't need to update this

Reply:
Reverted this change, thanks.